### PR TITLE
yamllint formatting cleanup for ocp-workload-optaweb-employee-rostering

### PR DESCRIPTION
**Pull request by ritesh97-rh**
##### SUMMARY
For the specified roles:
- Create a yamllint file to enforce formatting rules
- Make changes to existing code base to adhere to formatting requirements
- yamllint errors will be displayed during formatting checks in future development

##### ISSUE TYPE
- Code Cleanup Request

##### COMPONENT NAME
ocp-workload-optaweb-employee-rostering

### CODE REVIEW

File: ansible/roles/ocp-workload-optaweb-employee-rostering/tasks/remove_workload.yml [hunk 1]
```diff
@@ -5,14 +5,14 @@
 
 - name: define ocp_project
   set_fact:
-    ocp_project: "optaweb-employee-rostering-{{guid}}"
+    ocp_project: "optaweb-employee-rostering-{{ guid }}"
 
 - name: Remove user Project
-  shell: "oc delete project {{ocp_project}}"
+  shell: "oc delete project {{ ocp_project }}"
```
Comment (stencell) [inline ansible/roles/ocp-workload-optaweb-employee-rostering/tasks/remove_workload.yml:11]: Please choose to quote the commands or not and maintain consistency. See line 15 for no quote usage.

File: ansible/roles/ocp-workload-optaweb-employee-rostering/tasks/wait_for_deploy.yml [hunk 2]
```diff
@@ -15,16 +17,24 @@
...
-- name: "Wait for following deployments to become ready: {{pod_to_wait}}"
+- name: "Wait for following deployments to become ready: {{ pod_to_wait }}"
   command: 'oc get rc -o json -n "{{ ocp_project }}"'
   register: rc_state
   changed_when: false
   retries: "{{ deploy_status_retries }}"
   delay: "{{ deploy_status_delay }}"
-  until: 'rc_state.stdout |from_json |json_query("items[? ((status.readyReplicas == \"\" || status.readyReplicas == ''`0`'') && (metadata.annotations.\"openshift.io/deployment.cancelled\" != ''true''))].metadata.annotations.\"openshift.io/deployment-config.name\"") |intersect(pod_to_wait) |length == 0'
+  until: >-
+    c_state.stdout | from_json | json_query("items[?
```
Comment (stencell) [inline ansible/roles/ocp-workload-optaweb-employee-rostering/tasks/wait_for_deploy.yml:33]: This change cut out the var name at the beginning. It should be `rc...`.

**Review by stencell — CHANGES_REQUESTED** [review]
Fixes required and must be tested before merging.

File: ansible/roles/ocp-workload-optaweb-employee-rostering/tasks/remove_workload.yml (hunk 1 above)
Comment (ritesh97-rh) [reply ansible/roles/ocp-workload-optaweb-employee-rostering/tasks/remove_workload.yml:11]: Haven't done that anywhere - was not told that we are enforcing this. The files are originally mixed up in terms of usage, and some of them are hundreds of lines long. Will not get to doing this before my time on this ends.

File: ansible/roles/ocp-workload-optaweb-employee-rostering/tasks/wait_for_deploy.yml [hunk 3]
```diff
@@ -15,16 +17,24 @@
   changed_when: false
   retries: "{{ deploy_status_retries }}"
   delay: "{{ deploy_status_delay }}"
-  until: 'rc_state.stdout |from_json | json_query("items[? (kind == ''ReplicationController'')].metadata.annotations.\"openshift.io/deployment-config.name\"") |intersect(item) |length >= 1'
-  loop: "{{pod_to_wait}}"
+  until: >-
+    rc_state.stdout | from_json | json_query("items[?
+    (kind == ''ReplicationController'')].metadata.annotations."openshift.io/deployment-config.name"")
```
Comment (jkupferer) [inline ansible/roles/ocp-workload-optaweb-employee-rostering/tasks/wait_for_deploy.yml:22]: This `''` quote is not correct

**Review by jkupferer — CHANGES_REQUESTED** [review]

**Pull request closed without merging**